Resources/Locales/pt_br/openspades.po

```diff
@@ -544,7 +544,7 @@ msgstr "Ver."
 #: ../../Scripts/Gui/MainScreen.as:515 ../../Scripts/Gui/MainScreen.as:529
 msgctxt "MainScreen"
 msgid "Loc."
-msgstr "Loc."
+msgstr "Ling."
```

Comment:
@DarkDracoon "Loc." represents the physical location of the server, not the language.

Also, I am the translator of the pt_br language, thanks for the fixes. Submit a pull request whenever you find something that can be improved :)